scss/lessonCard.scss

```diff
@@ -0,0 +1,21 @@
+.card-title {
+  font-size: 1.5rem;
+  font-weight: bold;
+  line-height: 1.33;
+}
+.icon-messages {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 75%;
+  justify-content: space-between;
+  white-space: nowrap;
+}
+.icon-text {
+  font-size: 0.8125rem;
+  line-height: 1.38;
+}
+.lesson-description {
```

Comment:
according to [BEM](https://en.bem.info/methodology/quick-start/), this should be `.lesson-card__description`

Reply:
Corrected 👍